packages/mask/src/plugins/NextID/SNSAdaptor/index.tsx

```diff
@@ -1,5 +1,5 @@
 import type { Plugin } from '@masknet/plugin-infra'
-import { PersonaSelectPanelDialog } from '../../../components/shared/PersonaSelectPanelDialog'
+import { PersonaSelectPanelDialog } from '../../../components/shared/ConnectPersonaSelect/PersonaSelectPanelDialog'
```

Comment:
From my point, a component is an object since we should name it with a noun.

+ `ConnectPersona` VERBE + NONE ✅
+ `ConnectPersonaSelect` VERBE + NONE + VERBE ❌ `Didn't name with an noun`
+ `SelectPersonaDialog` VERBE + NONE + NONE ✅
+ `SelectPersonaPanelDialog`  VERBE + NONE + NONE + NONE ❌ `Not clear a panel component or a dialog component`